adapters/openweb/openweb.go

```diff
@@ -16,175 +20,115 @@ type adapter struct {
 	endpoint string
 }
 
-type openwebImpExt struct {
-	OpenWeb openrtb_ext.ExtImpOpenWeb `json:"openweb"`
-}
-
-func (a *adapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
-	totalImps := len(request.Imp)
-	errors := make([]error, 0, totalImps)
-	sourceIdToImpIds := make(map[int][]int)
-	var sourceIds []int
-
-	for i := 0; i < totalImps; i++ {
-
-		sourceId, err := validateImpression(&request.Imp[i])
-
-		if err != nil {
-			errors = append(errors, err)
-			continue
-		}
-
-		if _, ok := sourceIdToImpIds[sourceId]; !ok {
-			sourceIdToImpIds[sourceId] = make([]int, 0, totalImps-i)
-			sourceIds = append(sourceIds, sourceId)
-		}
-
-		sourceIdToImpIds[sourceId] = append(sourceIdToImpIds[sourceId], i)
-
+func (a *adapter) MakeRequests(request *openrtb2.BidRequest, _ *adapters.ExtraRequestInfo) (requestsToBidder []*adapters.RequestData, errs []error) {
+	org, err := checkExtAndExtractOrg(request)
+	if err != nil {
+		errs = append(errs, fmt.Errorf("checkExtAndExtractOrg: %w", err))
```

Comment:
based on `checkExtAndExtractOrg` implementation, looks like adapter expects to set same org value in `imp.Ext`.

@IgorKulemzin  could you confirm if this is an expected behavior? If yes then it should be explicitly mentioned in openweb PR doc

Reply:
@onkarvhanumante Yes, it expects either `org` or legacy `aid` parameter. We'll update the documentation after this PR is merged, of course.